### ISSUE TITLE
The following is an issue I encountered using Azure Cloud shell while trying to create a service fabric cluster. I noticed that a similar issue was reported by another user earlier. Please let us know if it was resolved. Thanks. 

### ISSUE THREAD

**Issue by rameshgit001** [Azure/azure-cli#12811]
## Describe the bug

**Command Name**
`az sf cluster create`

**Errors:**
```
'NoneType' object has no attribute 'get'
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 608, in execute
    raise ex
cli/core/commands/__init__.py, ln 666, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
...
cli/command_modules/servicefabric/custom.py, ln 1158, in _create_certificate
    cmd, cli_ctx, vault_resource_group_name, vault_name, location, enabled_for_deployment=True).result()
cli/command_modules/servicefabric/custom.py, ln 1667, in _create_keyvault
    permissions = Permissions(keys=[KeyPermissions.get,
AttributeError: 'NoneType' object has no attribute 'get'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az sf cluster create --resource-group {} --location {} --certificate-output-folder {} --certificate-password {} --certificate-subject-name {} --cluster-name {} --cluster-size {} --os {} --vault-name {} --vault-resource-group {} --vm-password {} --vm-user-name {}`

## Environment Summary
```
Linux-4.15.0-1075-azure-x86_64-with-debian-stretch-sid
Python 3.6.5

azure-cli 2.2.0 *

```

**Comment by a-santamaria** [comment]
thanks for reporting I'm closing it as its duplicate of #12470. should be fix with pr #12731

**Comment by rameshgit001** [comment]
Thanks a lot for looking into this. As of today, the issue is still happening though.
